src/utils/fetchData.js

```diff
@@ -0,0 +1,30 @@
+export default class apiActions {
+  static createGame = (URL_API) => fetch(`${URL_API}games/`, {
+    method: 'POST',
+    body: JSON.stringify({
+      name: 'my game one',
+    }),
+    headers: {
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((res) => res.json())
+    .then((data) => data.result.split(' ')[3])
+
+  static getData = (URL_API) => fetch(URL_API)
+    .then((res) => res.json())
+    .then((data) => data)
+
+  static sendData = (URL_API, user, score) => fetch(URL_API, {
+    method: 'POST',
+    body: JSON.stringify({
+      user,
+      score,
+    }),
+    headers: {
+      'Content-Type': 'application/json; charset=UTF-8',
+    },
+  })
+    .then((res) => res.json())
+    .then((json) => json)
+}
```

Comment:
- [ ]  Please, let's use `async` and `await` with the `fetch` method to send and receive data.
- [ ] Please, include only relevant information in your README. You have provided instructions on how to run tests that you have not implemented in your application. Please, let's fix this.

Reply:
Hi @george-shammar, 
Thank you for your time and thoughtful comments, i just want to point out the fact that  async  and await are used when the actions are actually called inside the module consumeApi.js. Could this be enough to consider this point as resolved?